src/ui/components/shared/Onboarding/BubbleModal.tsx

```diff
@@ -24,11 +24,11 @@ export function BubbleBackground() {
 export default function BubbleModal({ children }: { children: ReactNode }) {
   return (
     <div
-      className="w-full h-full grid fixed z-50 items-center justify-center"
+      className="w-full h-full grid fixed z-50 items-center justify-center relative"
```

Comment:
This made things wonky. Why was `relative` moved up??

Reply:
Ah it's the loading tips that are misaligned otherwise. Let me take a stab at it in a separate PR.